bundles/org.openhab.binding.http/src/main/java/org/openhab/binding/http/internal/HttpBindingConstants.java

```diff
@@ -23,8 +24,12 @@
  */
 @NonNullByDefault
 public class HttpBindingConstants {
-
-    public static final String BINDING_ID = "http";
+    private static final String BINDING_ID = "http";
 
     public static final ThingTypeUID THING_TYPE_URL = new ThingTypeUID(BINDING_ID, "url");
+
+    public static final ChannelTypeUID REQUEST_DATE_TIME_CHANNELTYPE_UID = new ChannelTypeUID(BINDING_ID,
+            "requestDateTime");
+    public static final String CHANNEL_LAST_SUCCESS = "lastSuccess";
+    public static final String CHANNEL_LAST_FAILURE = "lastFailure";
```

Comment:
The id's in the XML thing definition are now:
```suggestion
            "request-date-time");
    public static final String CHANNEL_LAST_SUCCESS = "last-success";
    public static final String CHANNEL_LAST_FAILURE = "last-failure";
```